README.md

```diff
@@ -100,3 +104,7 @@ Please refer to our [Code of Conduct][codeconduct].
 [codeconduct]: https://github.com/dragonflyoss/community/blob/master/CODE_OF_CONDUCT.md
 [d7y.io]: https://d7y.io/
 [dingtalk]: docs/images/community/dingtalk-group.jpeg
+
+## Software bill of materials
+
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fdragonflyoss%2Fdragonfly.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fdragonflyoss%2Fdragonfly?ref=badge_large)
```

Comment:
![image](https://github.com/user-attachments/assets/24ac9074-3df5-4ce6-80af-f04245a69cd4)
The badge already exists, there is no need to add this content.

Reply:
No, The requirement is as the following, it must be displayed as a paragraph rather than an icon: https://clomonitor.io/docs/topics/checks/#software-bill-of-materials-sbom